demo/server/src/app.ts

```diff
@@ -14,6 +14,16 @@ import appConfig from './util/config';
 
 const app: express.Express = express();
 const config = appConfig.getConfig();
+let corsConfig = {};
+if (!config.keycloakConfig) {
+  const dynamicOrigin = function(origin, callback) {
```

Comment:
This looks like critical part of the integration so let's extract that to separate method and make sure that we document what is happening here.